### ISSUE TITLE
TypeError: undefined is not a function

### ISSUE THREAD

**Issue by daleharvey** [tmpvar/jsdom#1097]
This started happening to me recently, if I copy the test script in the README

```
var jsdom = require("jsdom").jsdom;
var document = jsdom("hello world");
var window = document.defaultView;

console.log(window.document.documentElement.outerHTML);
// output: "<html><head></head><body>hello world</body></html>"

console.log(window.innerWidth);
// output: 1024

console.log(typeof window.document.getElementsByClassName);
// outputs: function
```
I get
```
$ node lib/test.js
/Users/daleharvey/src/arandomurl.com/node_modules/jsdom/lib/jsdom/browser/Window.js:41
  this._document = new dom.HTMLDocument({
                   ^
TypeError: undefined is not a function
    at new Window (/Users/daleharvey/src/arandomurl.com/node_modules/jsdom/lib/jsdom/browser/Window.js:41:20)
    at exports.jsdom (/Users/daleharvey/src/arandomurl.com/node_modules/jsdom/lib/jsdom.js:47:16)
    at Object.<anonymous> (/Users/daleharvey/src/arandomurl.com/lib/test.js:5:16)
    at Module._compile (module.js:428:26)
    at Object.Module._extensions..js (module.js:446:10)
    at Module.load (module.js:353:32)
    at Function.Module._load (module.js:308:12)
    at Function.Module.runMain (module.js:469:10)
    at startup (node.js:124:18)
    at node.js:882:3
```

**Comment by inikulin** [comment]
@daleharvey Which version of jsdom do you use? This should be fixed in >=5.0.0

**Comment by domenic** [comment]
I think I saw something similar and fixed it in 5.0.1 but was unable to write a test to verify at the time, possibly due to our testing infrastructure...

**Comment by inikulin** [comment]
@domenic Looks like issue occurred after jscs-related refactoring. Was fixed with this line: https://github.com/tmpvar/jsdom/blob/master/lib/jsdom.js#L19 But it should be already included in 5.0.0

**Comment by BerkeleyTrue** [comment]
I can confirm this issue exist in 4.5.2 but is fixed in 5.1.0. Maybe a patch to the 4.x branch is in order?

**Comment by domenic** [comment]
We generally don't backport. Just use 5.x.
